src/interfaces/oracles/IBeefyVeloVaultRelayer.sol

```diff
@@ -4,39 +4,19 @@ pragma solidity 0.8.20;
 import {IBaseOracle} from '@interfaces/oracles/IBaseOracle.sol';
 
 import {IBeefyVaultV7} from '@interfaces/external/IBeefyVaultV7.sol';
-import {IVeloPool} from '@interfaces/external/IVeloPool.sol';
-import {IPessimisticVeloLpOracle} from '@interfaces/external/IPessimisticVeloLpOracle.sol';
 
 interface IBeefyVeloVaultRelayer is IBaseOracle {
```

Comment:
is also `IAbstractVeloVaultRelayer` (so that when you use this interface you have `veloPool()` for example)